common/build-style/go.sh

```diff
@@ -41,8 +41,7 @@ do_build() {
 }
 
 do_install() {
-	find "${GOPATH}/bin" -type f -executable | while read line
-	do
-		vbin "${line}"
+	for f in "$(find "${GOPATH}/bin" -type f -executable)"; do
```

Comment:
remove the quotes around `$()` or it'll fail when it finds more than one binary

Reply:
oh right, sometimes I forgot about quoting...

Reply:
fixed, thank you.

Reply:
Is it possible/better to avoid find? Something like:
```sh
for f in "${GOPATH}"/**/*; do
    if [ -f "$f" ] && [ -x "$f" ]; then
        vbin "$f"
    fi
done
```
-----
I measured with my "/usr/lib", glob version is 6 or 7 times faster than find

Reply:
The higher number loop, the bigger different because the find version need a sub-shell and a fork-execve for find.